src/renderer.ts

```diff
@@ -156,6 +158,632 @@ export class DefaultCellRenderer implements ICellRendererFactory {
 /**
  * a renderer rendering a bar for numerical columns
  */
+
+class HeatmapCellRenderer extends DefaultCellRenderer {
+
+  createSVG(col: Column, context: IDOMRenderContext): ISVGCellRenderer {
+    function cell_dim(total, cells) {
+      return (total / cells);
+    }
+
+    const total_width = col.getWidth();
+    const bins = (<any>col.desc).datalength;
+    const defaultdomain = [0, 100];
+    var domain = (<any> col.desc).domain;
```

Comment:
try avoid accessing `desc` directly, properties should be wrapped within the model class